data_sources/sysmon_eventid_29.yml

```
@@ -0,0 +1,61 @@
+name: Sysmon EventID 29
+id: 06c61e04-2d07-4e85-bcd5-8110938b1b18
+version: 1
+date: '2025-11-14'
+author: Bhavin Patel, Splunk
```

I think its good to credit yourself lol
```suggestion
author: Teoderick Contreras, Splunk
```